content/en/docs/appstore/app-services/general-purpose-ocr.md

```diff
@@ -6,39 +6,191 @@ tags: ["Document Service", "AI", "ML", "OCR", "Industrial", "Manufacturing"]
 
 ## 1 Introduction
 
-The [General Purpose Optical Character Recognition](https://marketplace.mendix.com/link/component/118392) app service can help you extract text from images or PDF documents and get output in JSON and XML formats in bulk.
+The [General Purpose Optical Character Recognition](https://marketplace.mendix.com/link/component/118392) app service can help you extract text, tables, and barcodes from images or PDF documents and get output in XML formats in bulk.
+
+### 1.1 Features
+
+* Extract data(text,table and barcode) in XML format from images and pdf documents in bulk and map data to an entity.
+* Support [Mendix SSO](/appstore/modules/mendix-sso/)
+
+### 1.2 Limitation
+
+* Currently only supports images(in JPG, JPEG, PNG, BMP, TIFF formats) and Pdf documents. 
+* Total documents size cannot exceed 20 MB.
+* ABBYY FineReader Engine will not open images larger than 32512*32512 pixels.
```

Comment:
@VivekSinghRawat-1 
We need not mention the ABBYY FineReader limitation. We can mention as our service limitation